File: CS/OData.Server/BusinessObjects/OrderItem.cs
using System.ComponentModel;
using System.Reflection;
using System.Runtime.CompilerServices;
using System.Xml.Serialization;
using DevExpress.Persistent.BaseImpl.EF;

namespace OData.Server.BusinessObjects {
    public class OrderItem : BaseObject {
        public virtual string ProductName { get; set; }
        public virtual decimal UnitPrice { get; set; }
        public virtual int Quantity { get; set; }
        public virtual float Discount { get; set; }
        public virtual DateTime OrderDate { get; set; }

        public static List<OrderItem> GetInitialData() {
            var assembly = Assembly.GetExecutingAssembly();
            var resourceName = assembly.GetManifestResourceNames().Single(str => str.EndsWith("order_items.xml"));

            XmlSerializer xsSubmit = new XmlSerializer(typeof(List<OrderItem>));
            using(Stream stream = assembly.GetManifestResourceStream(resourceName))
                return (List<OrderItem>)xsSubmit.Deserialize(stream);
        }
    }
}

File: VB/OData.Server/BusinessObjects/OrderItem.vb
Imports System.ComponentModel
Imports System.Reflection
Imports System.Runtime.CompilerServices
Imports System.Xml.Serialization
Imports DevExpress.Persistent.BaseImpl.EF

Namespace OData.Server.BusinessObjects

    Public Class OrderItem
        Inherits BaseObject

        Public Overridable Property ProductName As String

        Public Overridable Property UnitPrice As Decimal

        Public Overridable Property Quantity As Integer

        Public Overridable Property Discount As Single

        Public Overridable Property OrderDate As DateTime

        Public Shared Function GetInitialData() As List(Of OrderItem)
            Dim assembly = Assembly.GetExecutingAssembly()
            Dim resourceName = assembly.GetManifestResourceNames().[Single](Function(str) str.EndsWith("order_items.xml"))
            Dim xsSubmit As XmlSerializer = New XmlSerializer(GetType(List(Of OrderItem)))
            Using stream As Stream = assembly.GetManifestResourceStream(resourceName)
                Return CType(xsSubmit.Deserialize(stream), List(Of OrderItem))
            End Using
        End Function
    End Class
End Namespace
